csm/conf/configure.py

```diff
@@ -136,6 +136,8 @@ def set_csm_endpoint(self):
         # Conf.set(const.CSM_GLOBAL_INDEX, const.AGENT_HOST, csm_host)
         if csm_protocol == 'https':
             Conf.set(const.CSM_GLOBAL_INDEX, const.CSM_DEBUG_MODE, 'false')
+        else:
+            Conf.set(const.CSM_GLOBAL_INDEX, const.CSM_DEBUG_MODE, 'true')
```

Comment:
Why is this required?

Reply:
We are considering empty default value for those keys whose value is depend on conf store. 
Here i can see we are adding "true" only based on value of conf store.  we must provide value for each key if it depends on conf store.

Reply:
The debug mode config is not required for CSM now. We can take this change inthe next sprint. 
For now it is fine.